tests/pyfunc/test_pyfunc_schema_enforcement.py

```diff
@@ -1362,6 +1369,61 @@ def predict(self, context, model_input, params=None):
         "datetime_param"
     ] == np.datetime64("2023-06-26 00:00:00")
 
+    # Test model serving
```

Comment:
Not a blocker, but we should consider splitting tests.

1. Harder to tell what fails and succeeds
2. More likely to make mistakes.
3. Harder to figure what's going on.
4. Takes longer to run.